cfme/tests/automate/test_file_import.py

```diff
@@ -31,3 +36,68 @@ def test_domain_import_file(import_datastore, import_data):
             3. Import should work. Check imported or not.
     """
     assert import_datastore.exists
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1753586])
+@pytest.mark.customer_scenario
+@pytest.mark.parametrize(
+    "import_data",
+    [
+        DatastoreImport("bz_1753586_user.zip", "bz_1753586_user", None),
+        DatastoreImport("bz_1753586_user_locked.zip", "bz_1753586_user_locked", None),
+        DatastoreImport("bz_1753586_system.zip", "bz_1753586_system", None),
+    ],
+    ids=["user", "user_locked", "system"],
+)
+def test_crud_imported_domains(import_data, temp_appliance_preconfig):
+    """
+    Bugzilla:
+        1753586
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        casecomponent: Automate
+        testSteps:
+            1. Export the appliance domain as a zip file
+            2. Extracted the ManageIQ directory to a new dir on the local machine called
+               Domain_Test02
+            3. From within the new Domain_Test02 dir, edit the __domain__.yaml file so that the
+               'name' attribute equals Domain_Test02
+               NOTE: Do not change the 'source' attribute from 'system' to 'user' this time.
+            4. Archived the new Domain_Test02 dir and upload to CloudForms Appliance
+        expectedResults:
+            1.
+            2.
+            3.
+            4. The imported domains should be Custom Domains or these imported domains should gets
+               deleted from db
+    """
+    # Download datastore file from FTP server
+    fs = FTPClientWrapper(cfme_data.ftpserver.entities.datastores)
+    file_path = fs.download(import_data.file_name)
+
+    # Import datastore file to appliance
+    datastore = temp_appliance_preconfig.collections.automate_import_exports.instantiate(
+        import_type="file", file_path=file_path
+    )
+    domain = datastore.import_domain_from(import_data.from_domain, import_data.to_domain)
+    assert domain.exists
+    view = navigate_to(domain.parent, "All")
+    if import_data.file_name == "bz_1753586_user.zip":
+        with update(domain):
+            domain.description = fauxfactory.gen_alpha()
+        domain.delete()
+        view.flash.assert_message(f'Automate Domain "{domain.description}": Delete successful')
+    elif import_data.file_name == "bz_1753586_user_locked.zip":
+        with update(domain):
+            domain.description = fauxfactory.gen_alpha()
+        domain.delete()
+        view.flash.assert_message(f'Automate Domain "{domain.description}": Delete successful')
```

Comment:
@ganeshhubale Since these `if, elif` blocks are doing the same thing, you can combine them.

Reply:
My bad!
Thanks @john-dupuy :+1: